pyflink-walkthrough/payment_msg_proccessing.py

```diff
@@ -73,9 +74,9 @@ def log_processing():
     t_env.register_function('province_id_to_name', province_id_to_name)
 
     t_env.from_path("payment_msg") \
-        .select("province_id_to_name(provinceId) as province, payAmount") \
-        .group_by("province") \
-        .select("province, sum(payAmount) as pay_amount") \
+        .select(call('province_id_to_name', col('provinceId')).alias("province"), col('payAmount')) \
+        .group_by(col('province')) \
+        .select(col('province'), call('sum', col('payAmount').alias("pay_amount"))) \
```

Comment:
That string syntax used before was deprecated and [removed in 1.16](https://issues.apache.org/jira/browse/FLINK-26986).